packages/thirdweb/src/wallets/create-wallet.ts

```diff
@@ -119,6 +119,7 @@ export function createWallet<const ID extends WalletId>(
             ] = await autoConnectInjectedWallet(
               id as InjectedSupportedWalletIds,
               emitter,
+              options.chain,
```

Comment:
likely also have to check that we pass this correctly into all the other wallets in `createWallet()`